src/main/java/nextstep/auth/application/service/TokenService.java

```diff
@@ -19,21 +18,21 @@
 @Service
 public class TokenService {
 
-    private final MemberRepository memberRepository;
+    private final UserDetailService userDetailService;
     private final JwtTokenProvider jwtTokenProvider;
     private final GithubClient githubClient;
 
 
     public TokenResponse createToken(String email, String password) {
-        Optional<Member> memberOptional = memberRepository.findByEmail(email);
-        if (memberOptional.isEmpty()) {
+        Optional<CustomUserDetail> userDetailOptional = userDetailService.findById(email);
+        if (userDetailOptional.isEmpty()) {
             throw new NotFoundException();
```

Comment:
```suggestion
        CustomUserDetail userDetail = userDetailService.findById(email).orElseThrow(NotFoundException::new);
```

이렇게 하면 31라인의 의도까지 포함되는 코드라고 생각합니다 😄

Reply:
감사합니다. 😄